src/Searcher.cpp

```diff
@@ -125,9 +131,10 @@ std::vector<Node> Searcher::expand(const Node& parent,
     return children;
 }
 
-void Searcher::move_order(std::vector<Node>& nodes) {
-    // Do nothing for now
-    // TODO: Better move ordering
+void Searcher::move_order() {
+    for (auto&& moves : ordered_moves) {
+        std::sort(moves.second.begin(), moves.second.end(), Searcher::ScoreSort());
```

Comment:
There is already a `Node::operator<` and `Node::operator>` that compares the scores. Use `std::greater<Node>()` or `std::less<Node>()`. Don't you also have to consider whether the current state is a `HOME` or an `AWAY`?